module/metrics/gossipsub_rpc_validation_inspector.go

```diff
@@ -14,13 +14,16 @@ type GossipSubRpcValidationInspectorMetrics struct {
 	prefix                                 string
 	rpcCtrlMsgInAsyncPreProcessingGauge    prometheus.Gauge
 	rpcCtrlMsgAsyncProcessingTimeHistogram prometheus.Histogram
+	invalidCtrlMessageErrorCountHistogram  prometheus.Histogram
```

Comment:
❗ this seems never used after the intialization.

Reply:
https://github.com/onflow/flow-go/pull/4978/commits/77530d00fece3ba71101a7898a0ae885e5f6caea